packages/client/hmi-client/src/petrinet/petrinet-renderer.ts

```diff
@@ -369,6 +380,10 @@ export class PetrinetRenderer extends BasicRenderer<NodeData, EdgeData> {
 		});
 
 		this.on('background-click', () => {
+			// Reset opacity from focus:
+			this?.chart?.selectAll('.node-ui').style('opacity', '1');
+			this?.chart?.selectAll('.edge').style('opacity', '1');
+
```

Comment:
Easier to read if you pass in numbers `0.3`, `1.0` or whatnot.

The optional-chaining is weird .... why do you have `this?`?